librosa/core/constantq.py

```diff
@@ -62,9 +63,9 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
```

Comment:
In that case, numpy array indexing will do the right thing.  For instance, say we have 36 bpo and n_bins = 12:

```python
In [2]: x = np.arange(12)

In [3]: x[-12:]
Out[3]: array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11])

In [4]: x[-36:]
Out[4]: array([ 0,  1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11])

```